src/UI/Component/Glyph/Factory.php

```diff
@@ -675,6 +675,57 @@ public function sad($action = null);
 	 */
 	public function angry($action = null);
 
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       The EyeClosed Glyph is used to toggle the revelation-mode of password fields.
+	 *   composition: >
+	 *       The EyeClosed Glyph uses the glyphicon-eye-close.
+	 *   effect: >
+	 *       When clicked, the password-field is masked, thus hiding the input.
+	 *
+	 * context: Used with password-fields to toggle mask/revealed mode.
+	 *
+	 * rules:
+	 *   composition:
+	 *       1: >
+	 *          The EyeClosed Glyph MUST only be used with Password-Inputs.
+	 *   accessibility:
+	 *       1: >
+	 *          The aria-label MUST be 'eye closed'.
```

Comment:
There should be some indication about the state by the aria attributes. I would prefer something like "Eye closed - charactors of inputs fields visible".